test/e2e/framework.go

```diff
@@ -61,9 +63,16 @@ const (
 	agentContainerName   string = "antrea-agent"
 	antreaYML            string = "antrea.yml"
 	antreaIPSecYML       string = "antrea-ipsec.yml"
+	antreaCovYML         string = "antrea-coverage.yml"
+	antreaIPSecCovYML    string = "antrea-ipsec-coverage.yml"
 	defaultBridgeName    string = "br-int"
 	monitoringNamespace  string = "monitoring"
 
+	antreaControllerCovBinary string = "antrea"
+	antreaAgentCovBinary      string = "antrea"
```

Comment:
I know that we use pgrep and it doesn't really matter, but why use "antrea" for both instead of "antrea-agent-coverage" and "antrea-controller-coverage" respectively?

Reply:
I met the below issue when using pgrep.  The pattern "antrea-controller-coverage"  or "antrea-controller" cannot be used to get the process pid.  The same for "antrea-agent-coverage" process.

root@antrea-linux-testbed6-0:~# kubectl exec -it  antrea-controller-5ddc79db6c-8m5vj -n kube-system bash
kubectl exec [POD] [COMMAND] is DEPRECATED and will be removed in a future version. Use kubectl kubectl exec [POD] -- [COMMAND] instead.
root@antrea-linux-testbed6-2:/# ps aux
USER       PID %CPU %MEM    VSZ   RSS TTY      STAT START   TIME COMMAND
root         1  0.0  0.0   2612   612 ?        Ss   02:18   0:00 /bin/sh -c antrea-controller-coverage -test.run=TestBincoverRunMain -test.coverprofile=antreacontroller.out -args-file=/cont
root         6  3.1  1.1 960340 48184 ?        Sl   02:18   0:01 antrea-controller-coverage -test.run=TestBincoverRunMain -test.coverprofile=antreacontroller.out -args-file=/controller-arg-
root        18  0.0  0.0   4244  3460 pts/0    Ss   02:19   0:00 bash
root        25  0.0  0.0   5884  2972 pts/0    R+   02:19   0:00 ps aux
root@antrea-linux-testbed6-2:/# pgrep antrea-controller-coverage
root@antrea-linux-testbed6-2:/# pgrep antrea-controller
root@antrea-linux-testbed6-2:/# pgrep antrea
6

Reply:
you could use `pgrep -f antrea-controller-coverage`. That's because command names in linux are limited to 15 chars.

Reply:
tried "pgrep -f" and this works! changed to grep the full name " antreaControllerCovBinary". Thanks a lot Antonin.